events.js

```diff
@@ -66,12 +66,22 @@ const events2019 = [
   }
 ];
 
-function initEventComponents() {
-  Vue.component('event-box', {
-    props: ['event'],
+const yearlyEvents = [
+  {
+    year: 2021,
+    events: events2021 
+  }, {
+    year: 2019,
+    events: events2019
+  }
+];
+
+function initGalleryComponents() {
+  Vue.component('gallery', {

```

Comment:
This component can be shared by events and concerts, so its name is changed to `gallery`.